Makefile

```diff
@@ -343,7 +345,7 @@ install-k8upv2:
 		--set k8up.envVars[5].name=BACKUP_GLOBALRESTORES3BUCKET,k8up.envVars[5].value=baas-restores \
 		--set k8up.envVars[6].name=BACKUP_GLOBALRESTORES3ACCESSKEYID,k8up.envVars[6].value=lagoonFilesAccessKey \
 		--set k8up.envVars[7].name=BACKUP_GLOBALRESTORES3SECRETACCESSKEY,k8up.envVars[7].value=lagoonFilesSecretKey \
-		--version=4.8.2 \
+		--version=4.8.4 \
```

Comment:
Also need to update the version in the CRD install step
https://github.com/uselagoon/lagoon-charts/blob/263cfad1632a442e9a5d202b2b7d4fb540b3daf4/Makefile#L332-L333

Maybe it is worth moving them to a variable, along with the others?